calicoctl get profile fails

Seen on master calicoctl.

```macncheese:vagrant casey$ calicoctl get profile -o wide
panic: template: get:2:47: executing "get" at <.Spec.Tags>: can't evaluate field Tags in type api.ProfileSpec

goroutine 1 [running]:
panic(0x1057b00, 0xc42036f640)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/projectcalico/calico-containers/calicoctl/commands.resourcePrinterTable.print(0x0, 0x0, 0x0, 0xc420092001, 0xc42039ebb0, 0x1, 0x1, 0x1, 0x1)
        /go/src/github.com/projectcalico/calico-containers/calicoctl/commands/printer.go:121 +0x410
github.com/projectcalico/calico-containers/calicoctl/commands.(*resourcePrinterTable).print(0xc4203c0a20, 0xc42039ebb0, 0x1, 0x1, 0x1, 0x0)
        <autogenerated>:25 +0x8a
github.com/projectcalico/calico-containers/calicoctl/commands.Get(0xc420266940, 0x4, 0x4)
        /go/src/github.com/projectcalico/calico-containers/calicoctl/commands/get.go:172 +0x3fc
main.main()
        /go/src/github.com/projectcalico/calico-containers/calicoctl/calicoctl.go:84 +0x600
```

--- a/calicoctl/resourcemgr/profile.go
+++ b/calicoctl/resourcemgr/profile.go
@@ -28,7 +28,7 @@ func init() {
 		[]string{"NAME", "TAGS"},
 		map[string]string{
 			"NAME": "{{.Metadata.Name}}",
-			"TAGS": "{{join .Spec.Tags \",\"}}",
+			"TAGS": "{{join .Metadata.Tags \",\"}}",
 		},
 		func(client *client.Client, resource unversioned.Resource) (unversioned.Resource, error) {
 			r := resource.(api.Profile)

